view/frontend/templates/button_product_page.phtml

```diff
@@ -139,10 +141,10 @@ if (!$block->isSupportableType()) return;
                      */
 
                     //check if we have inventory on stock
-                    if ( max_qty >= 0 && getQty() > max_qty ) {
+                    /*if ( max_qty >= 0 && getQty() > max_qty ) {
```

Comment:
I am thinking if we can save this check or not. Related asana: https://app.asana.com/0/0/1156147394609521/f